client/engine/messageservice/vectorclock_summarize_test.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 func TestSummarizeMessage(t *testing.T) {
+	t.Skip("TODO: Enable vector clock summarization with new message format")
```

Comment:
@geoknee Should we refactor the vectorclock message service to support the new message format or just remove it?

Reply:
I think it is fine to remove it, for now.

Reply:
Removed in https://github.com/statechannels/go-nitro/pull/864/commits/b6b4fb596ba02ff3159989fdda6cc290537a6aed